### ISSUE TITLE
[DemoApp] Add code example to show how to use token

### ISSUE THREAD

**Issue by ludovic35** [Orange-OpenSource/ouds-ios#283]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### As a

user of the Library

### ... I want to

see the code of a token usage

### ... so that I can

use it directly

### Motivation and context

See the design in figma.
All token screens should have this illustration 
![Image](https://github.com/user-attachments/assets/7788872e-8fa1-4690-b825-6aadbd2456f3)

**Comment by pylapp** [comment]
📣 **New TestFlight ALPHA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (@ludovic35 or @pylapp) to get access.

- Display name: **OUDS Showcase ALPHA (283)**
- Version: **0.7.0**
- Build number: **1732201889**
- Build tag: **19b5e5e**
- Build type: **ALPHA**
- Build details (GitHub): **283**


cc @B3nz01d (product owner)
cc @pylapp @ludovic35 @Tayebsed93 (dev team)
cc TonyAntonyHello @mccart77 @MaximeTonnerre (design team)

_This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by B3nz01d** [comment]
@Tayebsed93 When interacting with the "View token code example" to reduce it, when it is reduced it does not show the chevron anymore. 

The behavior should be this when opened
![Image](https://github.com/user-attachments/assets/4816335d-51d0-4edf-a6b0-b35defe90e08)

and this when closed:
![Image](https://github.com/user-attachments/assets/e03d7e20-ff95-432e-9977-16fe3bc2df4c)


And the default when you open the page should be with the closed version

Also the code exemple is not the token itself onle, it's how you use it in the code. ie: "theme.borderWidthDefault" or "theme.colorBgPrimary.color(for: colorScheme)" or "theme.spaceScaledMedium.dimension(for: horizontalSizeClass)"


And last but not least when the screen text size is raised to 190 the code title is changing but is not aligned anymore and the code part did not change size.

**Comment by pylapp** [comment]
@B3nz01d That build needs your review ;-)

**Comment by B3nz01d** [comment]
as described in the previous comment: "the default when you open the page should be with the closed version"
